pkg/option/config.go

```diff
@@ -1304,6 +1315,8 @@ func (c *DaemonConfig) Populate() {
 	c.KeepTemplates = viper.GetBool(KeepBPFTemplates)
 	c.KeepConfig = viper.GetBool(KeepConfig)
 	c.KVStore = viper.GetString(KVStore)
+	c.KVstoreLeaseTTL = viper.GetDuration(KVstoreLeaseTTL)
```

Comment:
We might want to consider also adding a validation snippet to `c *DaemonConfig) Validate()` to ensure that the duration is a "reasonable" value. That said I don't really know what a minimum viable TTL would be, I suspect it would need to be most likely longer than 10s, and possibly longer than that to prevent disruption of kvstore entries during restart. We'll see with the CI run whether 30s causes any trouble in the next couple of hours.

Anyone else have thoughts/opinions on this?

Reply:
Yeah, sounds good.
Also, should we also put an upper limit to kvstore LeaseTTL. Like etcd have an upper limit https://github.com/etcd-io/etcd/blob/master/clientv3/options.go#L64